scripts/release-actions.py

```diff
@@ -1,9 +1,14 @@
-import argparse
+import functools
```

Comment:
Hmm `#!/bin/python3` ?

Reply:
Better `#!/usr/bin/env python3` but why? It is not required if you follow `MAINTAINING.md` instructions.